package.json

```diff
@@ -28,12 +28,11 @@
     "rebuild-hard:dev": "npm run clean-slate && npm run build:dev",
     "rebuild-hard:prod": "npm run clean-slate && npm run build:prod",
     "precommit": "lint-staged",
-    "prepush": "lint-staged",
-    "prettier:base": "prettier --parser typescript --single-quote --trailing-comma es5 --tab-width 2 --no-semi --print-width 80 --write",
```

Comment:
I didn't get any errors, but no files were changed. I was using the CLI though, so maybe that's the issue. We can keep it in if it's working for you.